plugins/Marketplace/lang/en.json

```
@@ -147,6 +147,9 @@
         "AutoUpdateDisabledWarning": "You cannot install or update the plugin directly as automatic updates are disabled in the config. To enable automatic updates set %1$s in %2$s.",
         "ViewRepositoryChangelog": "View the changes",
         "Show": "Show",
-        "Sort": "Sort"
+        "Sort": "Sort",
+        "PluginLicenseStatusPending": "Your license key, for %1$s is in pending state. To resolve this, please go to your %2$sMarketplace account%3$s and update your payment details.",
+        "PluginLicenseStatusCancelled": "Your license key, for %1$s is cancelled. To continue using this plugin, go to your %2$sMarketplace%3$s account and start a new subscription for this plugin.",
```

I think the commas need to go.
```suggestion
        "PluginLicenseStatusPending": "Your license key for %1$s is in pending state. To resolve this, please go to your %2$sMarketplace account%3$s and update your payment details.",
        "PluginLicenseStatusCancelled": "Your license key for %1$s is cancelled. To continue using this plugin, go to your %2$sMarketplace%3$s account and start a new subscription for this plugin.",
```

---

@michalkleiner I will check with product and update you

---

@michalkleiner Updated as suggested :+1: